x-pack/test/functional/apps/ml/anomaly_detection_jobs/job_list.ts

```diff
@@ -96,51 +76,43 @@ export default function ({ getService }: FtrProviderContext) {
 
     after(async () => {
       await ml.api.closeAnomalyDetectionJob(jobId);
-      await ml.api.cleanMlIndices();
       await ml.api.deleteAnomalyDetectionJobES(jobId);
+      await ml.api.cleanMlIndices();
       await ml.testResources.deleteDataViewByTitle('ft_farequote');
     });
 
     it('expanded row with connected calendar shows link to calendar', async () => {
-      await ml.testExecution.logTestStep('calendar present for connected job');
-      await ml.expandedJobDetails.assertCalendarPresent(jobId, calendarId);
+      await ml.expandedJobDetails.assertJobRowCalendars(jobId, [calendarId]);
     });
 
     it('expanded row with forecast should display open forecast button', async () => {
-      await ml.testExecution.logTestStep('open forecast link present');
       await ml.expandedJobDetails.assertForecastElements(jobId);
     });
 
     it('expanded row with annotations can be edited', async () => {
-      await ml.testExecution.logTestStep('can edit annotations from expanded row');
-      await ml.expandedJobDetails.assertAnnotationsEdit(jobId);
+      await ml.expandedJobDetails.editAnnotation(jobId, 'appex qa rox');
     });
 
     it('expanded row with data feed flyout should display correctly', async () => {
-      await ml.testExecution.logTestStep(
-        'open data feed flyout and verify the title displays correctly'
-      );
       await ml.expandedJobDetails.assertDataFeedFlyout(jobId);
     });
 
     it('expanded row with model snapshot should display correctly', async () => {
-      await ml.testExecution.logTestStep(
-        'open model snapshot management and verify the action buttons are present'
-      );
       await ml.expandedJobDetails.assertModelSnapshotManagement(jobId);
     });
 
     it('multi-selection with one opened job should only present the opened job when job list is filtered by the Opened button', async () => {
       await ml.expandedJobDetails.selectAllJobs();
-      await ml.expandedJobDetails.assertColumnState('Opened', 'opened');
+      await ml.expandedJobDetails.assertJobListMultiSelectionText('2 jobs selected');
+      await ml.jobTable.filterByState(QuickFilterButtonTypes.Opened);
       await ml.expandedJobDetails.assertColumnId(jobId);
       await ml.expandedJobDetails.assertJobListMultiSelectionText('1 job selected');
     });
 
     it('multi-selection with one closed job should only present the closed job when job list is filtered by the Closed button', async () => {
-      await ml.expandedJobDetails.selectAllJobs();
-      await ml.expandedJobDetails.assertColumnState('Closed', 'closed');
+      await ml.jobTable.filterByState(QuickFilterButtonTypes.Closed);
       await ml.expandedJobDetails.assertColumnId(jobsSummaryId);
+      await ml.jobStatsBar.assertJobStatsBarItem('closed jobs', 1);
```

Comment:
IIRC the job stats bar doesn't change based on the job table filters. Would be good to double check. If it really doesn't change, this assertion doesn't tell us much (as it was one closed job before and after the filter.